### DISCUSSION TITLE
Please I am facing this problem and I want a solution

### DISCUSSION THREAD

**Discussion by wesamlibya** [projectdiscovery/nuclei#3689] [Q&A] [answered]
](url)

**Comment by wesamlibya** [comment]
How do I attach a picture

**Comment by wesamlibya** [comment]
[WRN] Found 5848 templates with syntax error (use -validate flag for further examination)
[INF] Using Nuclei Engine 2.8.5 (outdated)

**Comment by wesamlibya** [comment]
https://ibb.co/ZBWk9P4

**Comment by ehsandeep** [comment, marked as answer]
@wesamlibya see https://github.com/projectdiscovery/nuclei/discussions/3648

**Reply by wesamlibya** [reply]
Thank you brooo
